src/mk.rs

```diff
@@ -93,17 +93,29 @@ impl EncodingInfo for Multikey {
 
 impl From<Multikey> for Vec<u8> {
     fn from(val: Multikey) -> Self {
+impl Into<Vec<u8>> for Multikey {
+    fn into(self) -> Vec<u8> {
+impl From<Multikey> for Vec<u8> {
+    fn from(mk: Multikey) -> Vec<u8> {
```

Comment:
This looks incorrect. It looks like a merge/clippy --fix error.

Reply:
I thought I caught all these but looks like I missed some. When I did the merge something went wrong and it kept old and new copies. Sigh.

The [`from(value: T)`](https://doc.rust-lang.org/std/convert/trait.From.html#tymethod.from) is the idiomatic way we should be doing it to be consistent with the trait IMO, but it's a bikeshed color so I'm fine either way.

I'll go through these and clean them up

Reply:
These are marked outdated, did you make more commits? I'll take a look either way

Reply:
Yeah it looks like these dupes are from an older commit, the latest commit should be at https://github.com/cryptidtech/multikey/pull/2/commits/3b89dc2bc7b9b5fc1f6f53b1c29d2cc6d227d3ec

Reply:
You may have not pulled down those last couple of commits, which would make the difference

Reply:
oh shit...sorry...I didn't do a force push though so I wasn't aware that there were commits I needed to pull first. I'm sorry 'bout that.